src/Translator.php

```diff
@@ -4,7 +4,7 @@
 
 use ArrayAccess;
 use Illuminate\Database\Eloquent\Model;
-use TCG\Voyager\Facades\Voyager;
+//use TCG\Voyager\Facades\Voyager;
```

Comment:
The Voyager facade is used, so you cannot just remove it.

Reply:
Also, a pull request shouldn't claim something is unused and comment it out.  Git history exists for a reason, just remove the line.  In this case, however, you can't remove the line (or even comment it out) because it's needed like @marktopper said